releases/release-3.0.19.md

```diff
@@ -0,0 +1,52 @@
+---
+title: TiDB 3.0.19 Release Notes
+---
+
+# TiDB 3.0.19 Release Notes
+
+Release date: September 25, 2020
+
+TiDB version: 3.0.19
+
+## Compatibility Changes
+
++ PD
+
+    - Change the import path from `pingcap/pd` to `tikv/pd` [#2779](https://github.com/pingcap/pd/pull/2779)
+    - Change the copyright information from `PingCAP, Inc` to `TiKV Project Authors` [#2777](https://github.com/pingcap/pd/pull/2777)
+
+## Improvements
+
++ TiDB
+
+    - Mitigate the impact of failure recovery on QPS [#19764](https://github.com/pingcap/tidb/pull/19764)
+    - Support adjusting the concurrency on the `union` operator [#19885](https://github.com/pingcap/tidb/pull/19885)
+
++ TiKV
+
+    - Set `sync-log` to `true` always [#8636](https://github.com/tikv/tikv/pull/8636)
```

Comment:
```suggestion
    - Set `sync-log` to `true` by default [#8636](https://github.com/tikv/tikv/pull/8636)
```

Reply:
In fact, before this option is true by default, now we have removed this configuration item, so it always appears as true

Reply:
Thanks! So it is safe to say "set it to true permanently", right?